applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1057,26 +939,37 @@ static int initialize(le_audio_receive_cb recv_cb)
 		}
 	}
 
-#if !CONFIG_STREAM_BIDIRECTIONAL
-	struct bt_audio_unicast_group_stream_param group_params[ARRAY_SIZE(audio_streams)];
-	struct bt_audio_unicast_group_param param;
+	for (int i = 0; i < ARRAY_SIZE(stream_params); i++) {
+		if ((i % 2) == 0) {
+			stream_params[i].qos = &lc3_preset_sink.qos;
+			stream_params[i].stream = &headsets[headset_iterator].sink_stream;
+		} else {
+			stream_params[i].qos = &lc3_preset_source.qos;
+			stream_params[i].stream = &headsets[headset_iterator].source_stream;
+			headset_iterator++;
+		}
+	}
 
-	for (int i = 0; i < ARRAY_SIZE(audio_streams); i++) {
-		group_params[i].stream = &audio_streams[i];
-		group_params[i].qos = &lc3_preset_nrf5340.qos;
-		group_params[i].dir = BT_AUDIO_DIR_SINK;
+	for (int i = 0; i < ARRAY_SIZE(pair_params); i++) {
+		pair_params[i].tx_param = &stream_params[stream_iterator];
+		stream_iterator++;
+#if CONFIG_STREAM_BIDIRECTIONAL
+		pair_params[i].rx_param = &stream_params[stream_iterator];
+#else
+		pair_params[i].rx_param = NULL;
+#endif /* CONFIG_STREAM_BIDIRECTIONAL */
+		stream_iterator++;
 	}
 
-	param.params = group_params;
-	param.params_count = ARRAY_SIZE(audio_streams);
+	param.params = pair_params;
```

Comment:
Would appreciate an intro to the `pair_params` variable.

Reply:
Since we now can have bidirectional ISOs, a pair is basically a source stream and a sink stream for one ISO